scss/base/_variables.scss

```diff
@@ -5,4 +5,4 @@
   
 // Material icons related variables
 $material-icon-font-name: 'Material Icons';
-$material-icon-font-url: '../node_modules/material-design-icons/iconfont/';
+$material-icon-font-url: '../fonts/material-design-icons/iconfont/';
```

Comment:
We dont need the iconfont folder, place the fonts directly on the `material-design-icons` folder

Reply:
Done